src/examples/Cluster/Metrics/Samples.Cluster.AdaptiveGroup/Program.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Cluster.Metrics;
+using Akka.Cluster.Routing;
+using Akka.Configuration;
+using Samples.Cluster.Metrics.Common;
+
+namespace Samples.Cluster.AdaptiveGroup
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            var config = ConfigurationFactory.ParseString(await File.ReadAllTextAsync("Application.conf"));
+
+            var system = ActorSystem.Create("ClusterSystem", config);
+
+            var cluster = Akka.Cluster.Cluster.Get(system);
+            cluster.RegisterOnMemberUp(() =>
+            {
+                var paths = new List<string>
+                {
+                    "/user/factorialBackend-1",
+                    "/user/factorialBackend-2",
+                    "/user/factorialBackend-3",
+                    "/user/factorialBackend-4",
+                    "/user/factorialBackend-5",
+                    "/user/factorialBackend-6"
+                };
+
+                system.ActorOf(
```

Comment:
for the sample here, I would do two routers:

1. Created entirely from HOCON;
2. One created entirely from the fluent interface on `Props`.

So you can demonstrate how both work - and I would decorate them with comments.

Reply:
I did it on this line: https://github.com/akkadotnet/akka.net/pull/4691/files/e2847bd2e4fa084e3acc06fa7a97f7e895524421#diff-d1fee1dcd9811e796dd22889fe7ce472e977f115f35b914ba002783920119dedR45

Reply:
can you add some comments on it?

Reply:
Thinking about it from the point of view of an end-user who isn't sure what the syntax means right off the bat

Reply:
Will do